Neos.ContentRepository.Export/src/Event/ValueObject/ExportedEvent.php

```diff
@@ -7,39 +7,39 @@
 final class ExportedEvent implements \JsonSerializable
 {
     /**
+     * The Neos.ContentRepository.Core's domain events require the payload to be a json array string
+     * {@see \Neos\ContentRepository\Core\EventStore\EventInterface}
+     * This exporter will enforce this as well (by using array as payload type).
+     *
      * @param array<mixed> $payload
      * @param array<mixed> $metadata
      */
     public function __construct(
         public readonly string $identifier,
         public readonly string $type,
-        public readonly array $payload, // TODO: string
+        public readonly array $payload,
```

Comment:
fyi i tried to solve the todo https://github.com/neos/neos-development-collection/pull/4917 but its senseless ^^